test/models/work_test.rb

```diff
@@ -0,0 +1,19 @@
+require "test_helper"
+
+describe Work do
+  let(:work) { works(:one) }
+
+  # it "must be valid" do
+  #   expect(work).must_be :valid?
+  # end
+
+  it 'has required fields' do
+    fields = [:category, :title, :creator, :publication_year, :description ]
```

Comment:
You should also include the field `:votes`

You are also missing validation tests  for category, title, creator, publication_year and description.